src/Altinn.AccessManagement/Controllers/DelegationsController.cs

```diff
@@ -301,5 +301,48 @@ public async Task<ActionResult<List<DelegationExternal>>> GetAlInboundDelegation
                 return StatusCode(500);
             }
         }
+
+        /// <summary>
+        /// Endpoint for retrieving delegated resources between parties
+        /// </summary>
+        /// <response code="400">Bad Request</response>
+        /// <response code="500">Internal Server Error</response>
+        [HttpGet]
+        [Route("accessmanagement/api/v1/admin/delegations/maskinportenschema")]
+        [Authorize]
+        public async Task<ActionResult<List<MPDelegationExternal>>> GetAllDelegationsForAdmin([FromQuery] int? supplierOrg, int? consumerOrg, string scope)
```

Comment:
Organization numbers (as well as social security numbers) should be string input in order to handle those beginning with 0